server/src/modules-resolver.ts

```diff
@@ -11,7 +11,7 @@ import * as paths from 'path';
 export function findPathToModule(dir: string, moduleName: string): string|undefined {
     const stat = fs.statSync(dir)
     if (stat.isDirectory()) {
-        const candidate = paths.resolve(dir, 'node_modules', moduleName)
+        const candidate = require.resolve(moduleName, { paths: [dir] });
         if (fs.existsSync(candidate)) {
```

Comment:
`require.resolve` will throw if it can't find it so this check isn't needed, however this entire function can be replaced with 
```js
try {
  return require.resolve(moduleName, { paths: [dir] });
} catch {
  return undefined;
}
```